javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-player-app/index.html

```diff
@@ -142,19 +125,10 @@ <h2 class="playlist-title" id="playlist">Playlist</h2>
             <div></div>
             <div></div>
           </div>
-          <div>
-            <button
-              class="playlist-close"
-              aria-label="Playlist"
-              aria-expanded="true"
-            >
-              <div class="arrow"></div>
-            </button>
-          </div>
         </div>
-        <ul id="playlist-songs" aria-labelledby="playlist"></ul>
+        <ul id="playlist-songs"></ul>
       </div>
     </div>
-    <script src="script.js"></script>
   </body>
+  <script src="script.js"></script>
```

Comment:
Any reason why this was moved?

Reply:
Thanks for catching this. I think this was just an accident on our part.

I'll go ahead and move this back now to just before the closing `</body>` tag.